dcos/dcos-ui
# DCOS-57551: [UI] show a warning if a package has not been updated in a while

Add a last updated label for packages
that have not been updated soon.

Closes https://jira.mesosphere.com/browse/DCOS-57551

## Testing
1. Go to catalog tab.
2. Select a package that has been updated recently, for example cassandra and click it.
3. Verify that "Last Updated" label is shown on the package detail tab. The framework configuration tab (after clicking "Review & Run") should look the same.
4. Select a package that hasn't been updated for over a year, for example grafana and click it.
5. Verify that the "Last Updated" label is shown on the package detail tab. Next to it a warning icon with a tooltip should exist. The framework configuration tab (after clicking "Review & Run") should also have a warning icon with a tooltip.
6. Select a package with no last updated property, for example dcos-ui and click it.
7. Verify that the package detail page and the edit configuration page don't break.
8. Verify that the added tests makes sense.

## Trade-offs
See my comments.

## Dependencies
None.

## Screenshots
![OS Enterprise 2019-08-28 17-38-49](https://user-images.githubusercontent.com/300861/63870688-bc31ef80-c9ba-11e9-8c69-a42f8e3d7a07.png)
![Screenshot from 2019-08-28 14-20-12](https://user-images.githubusercontent.com/40791275/63851535-0efdae00-c99f-11e9-90f2-ceab3aa46312.png)
